CONST_versions.txt

```diff
@@ -62,6 +62,7 @@ socketpool==0.5.3
 SQLAHelper==1.0
 SQLAlchemy==0.9.8
 Tempita==0.5.2
+suds>=0.4
```

Comment:
You shouldn't modify this file, he will be overwritten on the c2cgeomapfish upgrade ...

You should add 
```
REQUIREMENTS += suds>=0.4
```
in the `geoportailv3.mk`